docker_containers_rm.sh

```diff
@@ -0,0 +1,2 @@
+docker container stop `docker container ps | grep dh-vis-* | awk '{print $1}'`
+docker container rm `docker container ls -a | grep dh-vis-* | awk '{print $1}'
```

Comment:
można też użyć docker-compose down, ale nie ma powodu żeby nie zostawić tego pięknego skryptu

Reply:
Mądrego to warto posłuchać.